src/components/Calendar/helpers/compareDates.js

```diff
@@ -0,0 +1,5 @@
+export default function compareDates(value1, value2) {
+    const date1 = new Date(value1).setHours(0, 0, 0, 0);
+    const date2 = new Date(value2).setHours(0, 0, 0, 0);
+    return Math.sign(date1 - date2);
```

Comment:
this is very clean but IE doesn't support Math.sign, since I saw that this is used again in order to not refactor too much I think you should create our own sign function like (getSign), don't forget test it.